_posts/2016-10-31-Kickstarters-Project.md

```diff
@@ -0,0 +1,205 @@
+This software would track the pull requests and merges automatically from Github and change the board accordingly.
+Continuous Integration was pretty important during the course of the project.
+This way whenever they made changes to the code and made a pull request to Github that failed to build, they had to fix their code before they could continue.
+Once the build on codeship succeeded and the pull request was merged. The main development branch and master branch would have their changes (if any) deployed to their Cloudfoundry instance.
+
+The process for the back-end was pretty simple.
+They started out with around 5 people, so some of them started to pair program while others started to program on their own.
+But, with paying attention to the SCRUM principles and how to write clean code.
+So starting with the basic implementations of the sensor, room and notifications classes and writing the JUnit and Mockito tests was their first job.
+While the back-end was pretty straight forward and relatively easy to begin with, the front-end team were confronted with some problems.
+The team consisted of only 2 people who had to tackle a lot of problems with the use of Angular 2 which was still in Beta at the time.
+Also the combination with the other frameworks wasn't quite that easy to work with.
+
+Every morning the team did a stand up meeting where they would discuss their changes in eachothers code and what they were going to do next.
+During these 4 weeks they tried to work in these short sprints but after a week or two it became clear this wouldn't be an easy task.
+A few team members already had left the group because they were assigned to projects, which messed up their sprints completely.
+
+During the first 2 weeks the back-end team started with using Spring Data JPA, but soon figured out Spring-boot-starter-data-MongoDB was the better alternative because of the large amount of data being pumped into the DB.
+A lot of time of went into providing Rest documentation that covers all the different calls handled by the application.
+This documentation was created with MockMvc tests, which create code snippets that are easy to use.
+
+On the front-end side they didn't have test cases yet, but nobody in the entire team had ever written front-end tests before.
+Which caused a little bit of a delay, also the webpages didn't seem to be responsive at all.
+Luckily there were some online tutorials at hand on mocking and writing front-end tests.
+The responsiveness issue was sovled by using their own components and CSS code instead of using material design.
+
+When starting their 2 final weeks there were only 3 people left who were able to fully commit to the dev-case.
+During the last week the kickstarters also had to prepare and give a proper introduction to the management of Ordina.
+Showing their newly learned presentation and introduction techniques.
+
+At the very end of the Kickstarters project the application was finished.
```

Comment:
I'd rephrase this as `... the core of the application was finished` since in the next section you mention that the project wasn't completely finished.